### PR TITLE
Added labelPosition and fullWidth props to Toggle

### PR DESCRIPTION
**Pull request by qubis741**
### Added

`Toggle`: Props `labelPosition` and `fullWidth`. ([@qubis741](https://https://github.com/qubis741) in [#2710](https://github.com/teamleadercrm/ui/pull/2710))

### Design
https://www.figma.com/file/Aq9Ovc5xBe93MfuPUHTNW6/New-Quotations---designs?type=design&node-id=10451-283721&mode=design&t=hPowg87zssp1apqC-0

### CODE REVIEW

File: src/components/toggle/theme.css [hunk 1]
```diff
@@ -29,10 +29,11 @@
     overflow: hidden;
     opacity: 0;
     margin: 0;
+    position: absolute;
```
Comment (lorgan3) [inline src/components/toggle/theme.css:32]: Q: shouldn't the toggle be `position: relative` then?

Comment (qubis741) [reply src/components/toggle/theme.css:32]: input is hidden. I am setting it to `absolute` so it wont mess with `justify-content: space-between`

**Comment by qubis741** [comment]
> Can we have a link to the design, please?

Added to description

File: src/components/toggle/theme.css (hunk 1 above)
Comment (lorgan3) [reply src/components/toggle/theme.css:32]: I see

**Review by lorgan3 — DISMISSED** [review]

**Comment by qubis741** [comment]
> Why not use the toggle with a `UIText` inside a `Flex`? All children of the flex container will stretch to full width by default. This is the implementation in the header customization menu of the new detail pages.

1. Its defined in Ahoy: https://ahoy.teamleader.design/3547946e7/p/812011-form-elements/b/40b1cb/t/281793
2. Its used atleast 3 times (project detail, quotation detail, now I need it in quote preview dialog)

**Comment by farazatarodi** [comment]
Thanks for the clarification. I think we can link Ahoy documentation to our changes to speed up the process in the future.

**Review by farazatarodi — DISMISSED** [review]

**Review by eniskraasniqi1 — APPROVED** [review]

**Review by farazatarodi — APPROVED** [review]

**Pull request merged**